i18n/fr/docusaurus-plugin-content-docs/current/additional.md

```diff
@@ -0,0 +1,64 @@
+---
+sidebar_position: 3000
+---
+
+# 🛸 Ressources supplémentaires
+
+## Ressources
+
+Une vidéo d'introduction fantastique à PE : https://youtube.com/watch?v=dOxUroR57xs&feature=shares
+
+Un très bon guide d'ingénierie de prompts courts : https://help.openai.com/en/articles/6654000-best-practices-for-prompt-engineering-with-openai-api
+
+Une excellente introduction à l'ingénierie de prompts : https://humanloop.com/blog/prompt-engineering-101
+
+Une belle collection de documents d'ingénierie de prompts : https://github.com/dair-ai/Prompt-Engineering-Guide
+
+De nombreux autres documents d'ingénierie de prompts : https://github.com/thunlp/PromptPapers
+
+Cours CMU sur l'ingénierie de prompts : https://youtu.be/5ef83Wljm-M
+
+Comment fonctionne Copilot : https://thakkarparth007.github.io/copilot-explorer/posts/copilot-internals.html
+
+Un guide de débutants pour l'écriture de prompts de Zapier : https://zapier.com/blog/gpt-3-prompt/
+
+Une liste de ressources impressionnante sur l'ingénierie des prompts de A à Z : https://github.com/promptslab/Awesome-Prompt-Engineering
+
```

Comment:
If you re-run the translation, note that the English has extra lines here now https://learnprompting.org/docs/additional